### DISCUSSION TITLE
CXXABI_1.3.11 not found

### DISCUSSION THREAD

**Discussion by frizzze** [lichess-org/fishnet#211] [Q&A] [answered]
What doesn't work if CXXABI_1.3.11 was not found?

Can I ignore these log entries of fishnet client? If not, how can I prevent it? 

```
[=|===============> 1 cores, 45 queued, latest: https://lichess.org/g9zhi4Zg/tmp/fishnet-QvDrHc/stockfish-x86-64-bmi2: /usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `CXXABI_1.3.11' not found (required by /tmp/fishnet-QvDrHc/stockfish-x86-64-bmi2)

E: Engine error: unexpected end of file
W: Worker 0 waiting for engine to shut down after error. Context: https://lichess.org/g9zhi4Zg#58
```

My debian has theses libraries installed:
```
libstdc++-6-dev:amd64               6.3.0-18+deb9u1
libstdc++6:amd64                    6.3.0-18+deb9u1
libstdc++6-alpha-cross              6.3.0-18cross1
```

$ ls -l /usr/lib/x86_64-linux-gnu/libstdc++.so.6*
```
lrwxrwxrwx 1 root root      19 Mon 10  2020 /usr/lib/x86_64-linux-gnu/libstdc++.so.6 -> libstdc++.so.6.0.22
-rw-r--r-- 1 root root 1566168 Mon 10  2020 /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.22

```

I hope this is the right place to ask. Didn't find better yet. Thanks in advance

**Comment by niklasf** [comment, marked as answer]
I assume your Debian has older system libraries than Ubuntu 18.04. Which release are you on?

Unfortunately it's awkward to make statically linked builds of Stockfish, and GitHub recently dropped support for their Ubuntu 16.04 build environment. So among the [changes from fishnet v2.5.0](https://github.com/niklasf/fishnet/releases/tag/v2.5.0) is:
> * Update system requirements from Ubuntu 16.04 to Ubuntu 18.04 or equivalent (due to glibc used by the workflow runner on GitHub).

The only real solutions are:
* Upgrading your distro (even if it would be perfectly fine to run and supported otherwise)
* [Building from source](https://github.com/niklasf/fishnet/blob/master/doc/install.md#from-source)
* [Using Docker](https://github.com/niklasf/fishnet/blob/master/doc/install.md#docker)

**Comment by frizzze** [comment]
Thank you Niklas,
I'm on Debian Stretch. And will try to build from source now :-)
